_data_templates/code_mandatory.yml

```diff
@@ -1,22 +1,27 @@
 --- 
+schemaVersion:
 adminCode: abc-def
 releases: 
   - 
     contact: 
-      URL: https://www.example.com
+      email: "abc@xyz.com"
+    date: 
+      created: "YYYY-MM-DD"
+      metadataLastUpdated: "YYYY-MM-DD"  
     description: 
       en: "Lorem ipsum dolor sit amet, consectetur adipiscing elit."
       fr: "Quisque bibendum ante ligula, porta vestibulum enim egestas vel."
     name: 
       en: "Example"
       fr: "Exemple"
-    permissions: 
-      licenses: 
-        - 
-          URL: 
-            en: "https://example.com/repo/example/LICENCE-en"
-            fr: "https://example.com/repo/example/LICENCE-fr"
-          name: "LICENCE"
+    licenses:
+      - 
+        URL: 
+          en: "https://example.com/repo/example/LICENCE-en"
+          fr: "https://example.com/repo/example/LICENCE-fr"
+        name: 
```

Comment:
SVP remplacer `name` ici par `spdxID` plutôt que d'ajouter le `fr` et `en`.

https://spdx.org/licenses/

Reply:
I replaced name by spdxID and I removed fr and en